skyplane/api/api.py

```diff
@@ -0,0 +1,262 @@
+"""API for the Skyplane object store"""
+import subprocess
+from functools import partial
+from pathlib import Path
+from shlex import split
+from tkinter import ON
+import traceback
+import uuid
+import os
+
+from rich import print as rprint
+
+import skyplane.cli
+import skyplane.cli.usage.definitions
+import skyplane.cli.usage.client
+from skyplane import GB
+from skyplane.cli.usage.client import UsageClient, UsageStatsStatus
+from skyplane.replicate.replicator_client import ReplicatorClient
+
+import typer
+from rich.progress import Progress, SpinnerColumn, TextColumn
+
+import skyplane.cli.cli_aws
+import skyplane.cli.cli_azure
+import skyplane.cli.cli_config
+import skyplane.cli.cli_internal as cli_internal
+import skyplane.cli.experiments
+from skyplane import cloud_config, config_path, exceptions, skyplane_root
+from skyplane.cli.common import print_header, console
+from skyplane.cli.cli_impl.cp_replicate import (
+    enrich_dest_objs,
+    generate_full_transferobjlist,
+    generate_topology,
+    confirm_transfer,
+    launch_replication_job,
+)
+from skyplane.cli.cli_impl.cp_replicate_fallback import (
+    replicate_onprem_cp_cmd,
+    replicate_onprem_sync_cmd,
+    replicate_small_cp_cmd,
+    replicate_small_sync_cmd,
+)
+from skyplane.replicate.replication_plan import ReplicationJob
+from skyplane.cli.cli_impl.init import load_aws_config, load_azure_config, load_gcp_config
+from skyplane.cli.common import parse_path, query_instances
+from skyplane.compute.aws.aws_auth_provider import AWSAuthenticationProvider
+from skyplane.compute.aws.aws_cloud_provider import AWSCloudProvider
+from skyplane.config import SkyplaneConfig
+from skyplane.config import _DEFAULT_FLAGS as config_default_flags
+from skyplane.obj_store.object_store_interface import ObjectStoreInterface
+from skyplane.utils import logger
+from skyplane.utils.fn import do_parallel
+import asyncio
+
+
+async def cp(
```

Comment:
Make a folder: `api/impl/cp_simple.py` and move this there. Do not expose it via `__init__.py`